source/rust_verify/src/rust_to_vir_base.rs

```diff
@@ -207,6 +207,8 @@ pub(crate) enum Attr {
     Trigger(Option<Vec<u64>>),
     // custom error string to report for precondition failures
     CustomReqErr(String),
+    // verify using bitvector theory, not converting back and forth with integer
```

Comment:
I'm not sure that this bit:

> , not converting back and forth with integer

will be clear to future readers.  Can we improve or remove it?

Reply:
Yes, I will just remove it. We will make sure the bit vector encoding is documented somewhere, along with a paper pencil proof of soundness.